docs/pages/project/contributing/contributing-docs.md

```diff
@@ -24,15 +24,101 @@ Meshery documentation is made of these components:
 
 ## Set up your development environment
 
-{% include alert.html type="info" title="Jekyll" content="The Meshery Docs site is built using Jekyll - a simple static site generator. You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)." %}
+{% include alert.html type="info" title="Jekyll" content="The Meshery Docs site is built using Jekyll - a simple static site generator. Jekyll can be installed in the different platforms like Windows, Linux, and MacOs by the following steps:" %}
```

Comment:
@debjani1413 let's not include a note like this with a trailing colon in an alert. The full message would need to be in the alert.

Heads-up on these items:

- MacOS - capitalize the "s"
- "can be installed on Windows, Linux, and MacOS" - - switch "in" and remove "the"
-